cif_pow.dic

```diff
@@ -2158,6 +2158,398 @@ save_pd_calib_d_to_tof.power
+    _name.linked_item_id          '_pd_meas.detector_id'
+    _type.purpose                 Link
+    _type.source                  Assigned
+    _type.container               Single
+    _type.contents                Code
+
+save_
+
+save_pd_calib_detected_intensity.detector_response
+
+    _definition.id
+        '_pd_calib_detected_intensity.detector_response'
+    _definition.update            2023-01-21
+    _description.text
+;
+    A value that indicates the relative sensitivity of each detector. This can
+    compensate for differences in electronics, size and collimation. Usually,
+    one detector, or the mean for all detectors, will be assigned the value
+    of 1.
+;
+    _name.category_id             pd_calib_detected_intensity
+    _name.object_id               detector_response
+    _type.purpose                 Measurand
+    _type.source                  Derived
+    _type.container               Single
+    _type.contents                Real
+    _enumeration.range            0.0:
+    _units.code                   none
+
+save_
+
+save_pd_calib_detected_intensity.detector_response_su
+
+    _definition.id
+        '_pd_calib_detected_intensity.detector_response_su'
+    _definition.update            2023-01-21
+    _description.text
+;
+    Standard uncertainty of _pd_calib_detected_intensity.detector_response.
+;
+    _name.category_id             pd_calib_detected_intensity
+    _name.object_id               detector_response_su
+    _name.linked_item_id
+        '_pd_calib_detected_intensity.detector_response'
+    _units.code                   none
+
+    _import.get                   [{'file':templ_attr.cif  'save':general_su}]
+
+save_
+
+save_pd_calib_detected_intensity.diffractogram_id
+
+    _definition.id
+        '_pd_calib_detected_intensity.diffractogram_id'
+    _definition.update            2023-01-21
+    _description.text
+;
+    A code which identifies the particular diffractogram from which this
+    intensity calibration was taken, if it was calibrated by a specimen.
+;
+    _name.category_id             pd_calib_detected_intensity
+    _name.object_id               diffractogram_id
+    _name.linked_item_id          '_pd_diffractogram.id'
+    _type.purpose                 Link
+    _type.source                  Assigned
+    _type.container               Single
+    _type.contents                Text
+
+save_
+
+save_pd_calib_detected_intensity.id
+
+    _definition.id                '_pd_calib_detected_intensity.id'
+    _definition.update            2023-01-21
+    _description.text
+;
+    A code to uniquely identify each intensity calibration.
+;
+    _name.category_id             pd_calib_detected_intensity
+    _name.object_id               id
+    _type.purpose                 Key
+    _type.source                  Assigned
+    _type.container               Single
+    _type.contents                Text
+    _enumeration.default          .
+
+save_
+
+save_pd_calib_detected_intensity.phase_id
+
```

Comment:
I think we need both `phase_id` and `diffractogram_id` as the diffractogram may have signal from multiple phases. So, all OK.

Reply:
There already are both phase and diffractogram ids.